app/views/steps/evidence/upload/_uploaded_file.html.erb

```diff
@@ -0,0 +1,15 @@
+<tr class="govuk-table__row">
+  <td id="<%=document.id%>" class="govuk-table__cell">
+    <span><%= document.filename %></span>
+
+    <% if document.s3_object_key %>
+      <strong class="govuk-tag govuk-tag--green">Uploaded</strong>
```

Comment:
If you add thee suggestion in the translation file, then we can do:
`      <strong class="govuk-tag govuk-tag--green"><%= t('.uploaded') %></strong>
`